aws-java-workshop/src/main/java/com/emc/vipr/s3/sample/_08_CreateLargeObject.java

```diff
@@ -20,10 +20,12 @@
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.InputStreamReader;
-import java.util.ArrayList;
-import java.util.List;
+import java.lang.reflect.Array;
+import java.util.*;
+import java.util.concurrent.*;
 
 public class _08_CreateLargeObject {
+	protected final AmazonS3 client = AWSS3Factory.getS3Client();
```

Comment:
It looks like this should be the client instance you use throughout your examples, but I see another instance named s3 that's also getting created and used.  I think you can change them to all use the global client reference.